explorer/client/src/components/transaction-card/RecentTxCard.tsx

```diff
@@ -114,6 +116,7 @@ function LatestTxView({
                         )}
                     >
                         <div className={styles.txcardgridlarge}>TxId</div>
+                        <div className={styles.txage}>Age</div>
```

Comment:
Can we label this "Time" or "Timestamp" instead (and change the associated variables)?
pictured: etherscan's version
<img width="909" alt="image" src="https://user-images.githubusercontent.com/14057748/175437264-8dbe48f3-c406-4b91-9ae4-a72baa5d50b2.png">

Reply:
Etherscan uses the term 'Age' when aggregating Transaction Results (https://etherscan.io/txs) and the term 'Timestamp' when it comes to a Transaction Results Page (above). I am ambivalent regarding which labels we use for the time.

@bwann52 --> How should we label the time field across the website?

Reply:
Following feedback on Slack from @bwann52 all labels for the timestamp field have been changed to `Time`